File: prakticke/bonus02/java/Rodinka.java
import java.util.Map;
import java.util.HashMap;

class Rodinka {
    public static class Riesenie {
        String otec;
        String matka;
        String syn;
        String dcera;
        public Riesenie(String o, String m, String s, String d) {
            otec = o; matka = m; syn = s; dcera = d;
        }
    }
    public Riesenie vyries() {
        return new Riesenie("", "", "", "");
    }
}

File: prakticke/bonus02/python/Rodinka.py
#! /usr/bin/env python3

class Rodinka:
    class Riesenie:
        def __init__(self, otec : str, matka : str, syn : str, dcera : str) -> None:
            self.otec = otec # type: str
            self.matka = matka # type: str
            self.syn = syn # type: str
            self.dcera = dcera # type: str
    def vyries(self) -> Riesenie:
        return Rodinka.Riesenie('', '', '', '')
